packages/sdk/src/utils/discovery.ts

```diff
@@ -16,8 +20,14 @@ const DEFAULT_NODE_REQUIREMENTS = {
 export function defaultPeerDiscoveries(
   pubsubTopics: PubsubTopic[]
 ): ((components: Libp2pComponents) => PeerDiscovery)[] {
+  // TODO: add a check to see if it is indeed TWN or if it is a custom network
+  // https://github.com/waku-org/js-waku/issues/2014
+  const dnsFleet = pubsubTopics.includes(DefaultPubsubTopic)
```

Comment:
@adklempner would this help with the issue we were looking at yesterday in the testing app?

Reply:
I guess this PR doesn't work as it didn't introduce any ENR to https://github.com/waku-org/js-waku/blob/master/packages/core/src/lib/predefined_bootstrap_nodes.ts (noticed by Jakub)

Reply:
All good, I forgot the fact that predefined nodes are not used and only ENR is used for discovery, which I added and it should work.